Documentation/development-flow.md

```diff
@@ -52,6 +52,65 @@ git remote -v
 ```
 Now you should have at least `origin` and `upstream` remotes. You can also add other remotes to collaborate with other contributors.
 
+## Layout
+
+A source code layout is shown below, annotated with comments about the use of each important directory:
+
+rook  
+├── build  
+├── cluster  
+│   ├── charts                    # Helm charts  
+│   │   └── rook-ceph  
+│   └── examples                  # Sample yaml files for rook cluster  
```

Comment:
Rook should be capitalized in all English prose/sentences.  It should only be lowercase when needed in code or manifests.